LAMP/DeleteContact.php

```diff
@@ -0,0 +1,36 @@
+<?php
+	$inData = getRequestInfo();
+
+	$conn = new mysqli("localhost", "TheBeast", "WeLoveCOP4331", "COP4331");
+	if ($conn->connect_error)
+	{
+		returnWithError( $conn->connect_error );
+	}
+	else
+	{
+		$stmt = $conn->prepare("DELETE from Contacts where UserID =? AND ContactID =?");
+		$stmt->bind_param("s", $inData["UserID"], $inData["ContactID"]);
```

Comment:
This is the weird thing about php where we have to declare what type of params we are passing through. Ex: s for string, i for int, d for double, etc. This took me a minute to figure out so no sweat
```suggestion
		$stmt->bind_param("ss", $inData["UserID"], $inData["ContactID"]);
```